src/devices/cpu/tlcs870/tlcs870.h

```diff
@@ -82,37 +84,19 @@ class tlcs870_device : public cpu_device
 		COND_F
 	};
 
-	// construction/destruction
-	tlcs870_device(const machine_config &mconfig, device_type type, const char *tag, device_t *owner, uint32_t clock, address_map_constructor program_map);
-
-	// device-level overrides
-	virtual void device_start() override;
-	virtual void device_reset() override;
-
-	// device_execute_interface overrides
-	virtual uint32_t execute_min_cycles() const override { return 2; }
-	virtual uint32_t execute_max_cycles() const override { return 26; }
-	virtual uint32_t execute_input_lines() const override { return 1; }
-	virtual bool execute_input_edge_triggered(int inputnum) const override { return inputnum == INPUT_LINE_NMI; }
-	virtual void execute_run() override;
-	virtual void execute_set_input(int inputnum, int state) override;
-
-	// device_memory_interface overrides
-	virtual space_config_vector memory_space_config() const override;
-
-	// device_state_interface overrides
-	virtual void state_string_export(const device_state_entry &entry, std::string &str) const override;
-	virtual void state_import(const device_state_entry &entry) override;
-	virtual void state_export(const device_state_entry &entry) override;
-
-	// device_disasm_interface overrides
-	virtual std::unique_ptr<util::disasm_interface> create_disassembler() override;
 
-	void tmp87ph40an_mem(address_map &map);
-
-	uint32_t m_debugger_temp;
+	enum _srcdst_addressingmode
```

Comment:
Since we've been talking about consistency, could you please pick a style and stick with it for `enum` declarations in this header?  The ones above have the opening brace on the same line as the declaration.  Either make them match this one, or make this one match the others.